### PR TITLE
Add year end review page

### PR DESCRIPTION
**Pull request by zhx828**
This fixes https://github.com/oncokb/oncokb/issues/3348

### CODE REVIEW

File: src/main/webapp/app/pages/yearEndReviewPage/BiomarkerTableData.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
+import React from 'react';
```
Comment (zhx828) [inline src/main/webapp/app/pages/yearEndReviewPage/BiomarkerTableData.tsx:1]: No need to review this file.

File: src/main/webapp/app/config/constants.tsx [hunk 2]
```diff
@@ -47,6 +47,8 @@ export const APP_WHOLE_NUMBER_FORMAT = '0,0';
 export const APP_TWO_DIGITS_AFTER_POINT_NUMBER_FORMAT = '0,0.[00]';
 export const NEWS_DATE_FORMAT = 'MMDDYYYY';
 export const NEWS_TITLE_DATE_FORMAT = 'MMMM D, YYYY';
+export const YEAR_END_REVIEW_DATE_FORMAT = 'YYYY';
+export const YEAR_END_REVIEW__TITLE_DATE_FORMAT = 'YYYY';
```
Comment (calvinlu3) [inline src/main/webapp/app/config/constants.tsx:51]:
```suggestion
export const YEAR_END_REVIEW_TITLE_DATE_FORMAT = 'YYYY';
```

File: src/main/webapp/app/index.scss [hunk 3]
```diff
@@ -110,7 +110,8 @@ b {
   }
 }
 
-.news {
+.news,
+.yearEndReview {
```
Comment (calvinlu3) [inline src/main/webapp/app/index.scss:114]: I think we use `year-end-review` format?

**Review by calvinlu3 — APPROVED** [review]

File: src/main/webapp/app/index.scss (hunk 3 above)
Comment (zhx828) [reply src/main/webapp/app/index.scss:114]: Yes in the scss file, but if it's in the modules.scss file, we do camel case. Let me fix.

**Review by scarrero4660 — APPROVED** [review]

**Pull request merged**